pkg/domain/login/auth_test.go

```diff
@@ -64,15 +52,12 @@ func validateToken(t *testing.T, tokenString string, accountID string, tokenSecr
 		return []byte(tokenSecret), nil
 	})
 	if err != nil {
-		t.Errorf("expected no error but got '%v'", err)
+		t.Fatalf("expected no error but got '%v'", err)
 	}
 	claims := token.Claims.(*jwt.RegisteredClaims)
-	if claims.Subject != accountID {
-		t.Errorf("expected '%s' but got '%s'", accountID, claims.Subject)
-	}
-	if claims.ID == "" {
-		t.Error("expected not empty id")
-	}
+	assert.Equalf(t, claims.Subject, accountID, "expected '%s' but got '%s'", accountID, claims.Subject)
```

Comment:
Aqui também dá pra ser bem mais resumido, algo assim:
```suggestion
	assert.Equal(t, accountID, claims)
```

O Equal vai passar por parâmetro: o test, o resultado desejado e o que retornou de fato (nesta ordem). E já vai comparar, mostrando na execução do teste se ocorrer erro o que era esperado e o que foi recebido.